Tests/LibVideo/TestVP9Decode.cpp

```diff
@@ -9,14 +9,15 @@
 #include <LibVideo/MatroskaReader.h>
 #include <LibVideo/VP9/Decoder.h>
 
-TEST_CASE(webm_in_vp9)
+inline void decode_video(StringView path, size_t expected_frame_count)
```

Comment:
No need for the `inline`.

Reply:
I figured this was the simplest way to make it compile without a declaration. I'll change it to a declaration and implementation.

Reply:
Does `static` not work?

Reply:
ah, I didn't realize that was valid outside of a scope, I'll change this then.